python/shotgun_desktop/errors.py

```diff
@@ -143,14 +143,3 @@ def __init__(self, var_name, path):
         )
 
 
-class InvalidAppStoreCredentialsError(ShotgunDesktopError):
```

Comment:
Remove "old" way (as in this morning's pull request :p) of handling the error for pre-zero-config desktop.